src/api/apiClients/axisApiClient.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosRequestConfig } from "axios";
```

Comment:
I believe it a typo in file name ('o' is missed in axios)

Reply:
Fixed